.github/workflows/build_check.yml

```diff
@@ -0,0 +1,20 @@
+name: Build Check CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the discovery firmware
+      run: make disco -C firmware
+    - name: Build the nucleo firmware
+      run: make disco -C firmware
```

Comment:
Typo

Reply:
thanks

Reply:
sorry this should be a draft
I'm experimenting rn

Reply:
Np thanks for setting this up

Reply:
np ready for review